pkg/minishift/util/input.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
what's about moving this to _pkg/util_? I have the following reasoning. _pkg/minishift/util_ is for utility functions which are specific to minishift. They don't really make sense in another context. _pkg/util_ are utilities which are generic. They are handy to have and I for example would like to take them with me to another project. 

I know, the separation is not quite like this, but I am in the process of doing a bit of a cleanup there. _homedir.go_ for example should move as well. Feel free to do this is part of this pull request as well. :-)